src/components/Ui/Input.vue

```diff
@@ -23,8 +25,22 @@ function handleInput(e) {
 </script>
 
 <template>
-  <UiButton
-    class="text-left w-full mb-2 flex px-3"
+  <div
+    class="
+      border border-skin-border
```

Comment:
Maybe we should we move this into a class f.e `fake-button` inside `style.css` so we can reuse in other components.